src/app/components/product-create/product-create.component.ts

```diff
@@ -8,32 +9,59 @@ import { fileValidator } from "../../validators/file-validator";
   templateUrl: './product-create.component.html',
   styleUrl: './product-create.component.scss'
 })
-export class ProductCreateComponent {
+export class ProductCreateComponent implements OnInit {
 
   productForm: FormGroup;
   formSubmitted: boolean = false;
   isLoading: boolean = false;
-
+  isEditMode: boolean = false;
+  productId: string | null = null;
   productName = '';
 
-  constructor(fb: FormBuilder, private productService: ProductService) {
-    this.productForm = fb.group({
-      name: ['a', Validators.required],
-      price: ['2', [Validators.required, Validators.pattern('^[0-9]*$')]],
-      description: ['b', Validators.required],
-      image: ['', [Validators.required, fileValidator(['image/jpeg', 'image/png'], 2 * 1024 * 1024)]],
+  constructor(
+    private fb: FormBuilder,
+    private productService: ProductService,
+    private route: ActivatedRoute,
+    private router: Router
+  ) {
+    this.productForm = this.fb.group({
+      name: ['', Validators.required],
+      price: ['', [Validators.required, Validators.pattern('^[0-9]*$')]],
+      description: ['', Validators.required],
+      image: ['', fileValidator(['image/jpeg', 'image/png'], 2 * 1024 * 1024)]
```

Comment:
are '.jpg' files included under the 'jpeg' extension?

Reply:
Yes, the '.jpg' are included in '.jpeg'